<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>901</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Image/background.jpg);
color:#fff;</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>561</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;
border-radius:16px;
background:rgba(0,0,0,0.3);
color:#fff</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:none;
background-image:none;
image:none;
background:rgba(255,255,255,0);</string>
     </property>
     <property name="text">
      <string>公司名称</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="register_company_name">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
border: 1px solid #4288CE;
border-image:none;
color:#fff;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="company_queren_pwd">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>230</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
border: 1px solid #4288CE;
border-image:none;
color:#fff;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="company_email">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
border: 1px solid #4288CE;
border-image:none;
color:#fff;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgba(255,255,255,0);</string>
     </property>
     <property name="text">
      <string>确认密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgba(255,255,255,0);</string>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgba(255,255,255,0);</string>
     </property>
     <property name="text">
      <string>邮箱</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="company_pwd">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>190</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
border: 1px solid #4288CE;
border-image:none;
color:#fff;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="company_xingming">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>150</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
border: 1px solid #4288CE;
border-image:none;
color:#fff;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgba(255,255,255,0);</string>
     </property>
     <property name="text">
      <string>姓名</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_zhuce">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>290</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgba(154, 207, 206,0.5);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;

</string>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_quxiao">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>290</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgba(154, 207, 206,0.5);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;
</string>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="company_queren_email">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
border: 1px solid #4288CE;
border-image:none;
color:#fff;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgba(255,255,255,0);</string>
     </property>
     <property name="text">
      <string>确认邮箱</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_isVersion">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>190</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>眼睛</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_isVersion_queren">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>230</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>眼睛</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>891</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:24px;
border-image:none;
font-weight:600;</string>
    </property>
    <property name="text">
     <string>注册新公司账户</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="register_company_message">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>480</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 170, 127);
border-image:none;</string>
    </property>
    <property name="text">
     <string>2682178536@qq.com客户QQ：2682178536</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
